### PR TITLE
Pop up covert address modal to covert user-input address to checksum address. fix #560

### LINKED ISSUE #560: Readonly address: if user input wrong checksum address, rwallet should alert and auto-convert the address
## Readonly wallet:
Title: Invalid RSK wallet address
Body: 
The entered wallet address is an invalid RSK address. RWallet will try to auto-correct it into the expected address format. Do you want to proceed?

Why are you seeing this message? (the same, in blue color)

Buttons:
Okay
Cancel

## Add custom token:
Title: Invalid RSK contract address
Body: 
The entered contract address is an invalid RSK address. RWallet will try to auto-correct it into the expected address format. Do you want to proceed?

Why are you seeing this message? (the same, in blue color)

Buttons:
Okay
Cancel

### PR DESCRIPTION
**Pull request by imstar15**
**Convert  address**

image

### CODE REVIEW

File: src/common/translations/pt-BR.json [hunk 1]
```diff
@@ -630,6 +631,14 @@
         "normal": "O endereço inserido não tem registro em %{explorerName} e tem %{balance} %{otherSymbolName} na rede RSK %{otherType}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK da %{type}?",
         "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK na %{type}?"
       }
+    },
+    "convertWalletAddress": {
+      "title": "Endereço de carteira RSK inválido",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:636]:
```suggestion
      "title": "Endereço RSK inválido",
```

File: src/common/translations/pt-BR.json [hunk 2]
```diff
@@ -630,6 +631,14 @@
...
+      "body": "O endereço da carteira inserido é um endereço RSK inválido. O RWallet tentará corrigi-lo automaticamente para o formato de endereço esperado. Você quer prosseguir?"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:637]:
```suggestion
      "body": "O endereço RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
```

File: src/common/translations/pt-BR.json [hunk 3]
```diff
@@ -630,6 +631,14 @@
...
+    },
+    "convertContractAddress": {
+      "title": "Endereço de contrato RSK inválido",
+      "body": "O endereço do contrato inserido é um endereço RSK inválido. O RWallet tentará corrigi-lo automaticamente para o formato de endereço esperado. Você quer prosseguir?"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:641]:
```suggestion
      "body": "O endereço do contrato RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Changes Portuguese Brazilian

**Review by chrisli30 — APPROVED** [review]
The refactor looks good 👍

**Pull request merged**